app/packages/spaces/src/AppModules.tsx

```diff
@@ -113,6 +114,11 @@ export function useOutsideClick() {
   // do nothing
 }
 
+export function useDimensions() {
+  const ref = useRef();
+  return { ref };
+}
+
```

Comment:
This feels like trivial code. Is it necessary?

Reply:
This is for mock app of spaces. Mock spaces does not import actual module, instead it mocks them in this file. Can be ignored